src/main/java/Duke.java

```diff
@@ -1,10 +1,214 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    // Printables
+    private static final String TAB = "    ";
+    private static final String LINE = String
+            .format("%s%s", TAB, "____________________________________________________________");
+    private static final String WELCOMEMESSAGE = String
+            .format("Hello! I'm Duke\n%s What can I do for you?", TAB);
+    private static final String ENDMESSAGE = "Bye. Hope to see you again soon!";
+    private static final String CREATEMESSAGE = "Got it. I've added this task:";
+    private static final String MARKMESSAGE = "Nice! I've marked this task as done:";
+    private static final String UNMARKMESSAGE = "OK, I've marked this task as not done yet:";
+    private static final String LISTMESSAGE = "Here are the tasks in your list:";
+    private static final String DELETEMESSAGE = "Noted. I've removed this task:";
+
+    // Commands
+    private static final String ENDCOMMAND = "bye";
+    private static final String PRINTCOMMAND = "list";
+    private static final String MARKCOMMAND = "mark";
+    private static final String UNMARKCOMMAND = "unmark";
+    private static final String TODOCOMMAND = "todo";
+    private static final String DEADLINECOMMAND = "deadline";
+    private static final String EVENTCOMMAND = "event";
+    private static final String DELETECOMMAND = "delete";
+
+    // Data structures
+    private static final List<Task> list = new ArrayList<>();
```

Comment:
Since `list` is used to store a few instances of `Task`, consider using plural forms to describe what is it you're storing in the list:
`Plural form should be used on names representing a collection of objects.`